src/schema/schema.graphql

```diff
@@ -309,9 +309,9 @@ type Hosts {
 
 "Structured representation of a tag"
 type StructuredTag {
-    namespace: String,
-    key: String!,
-    value: String
+    namespace: FilterStringWithWildcardWithLowercase,
```

Comment:
I don't think these changes aren't needed. When I run the code with this change I get this error: `The type of StructuredTag.value must be Output Type but got: FilterStringWithWildcardWithLowercase.`

Reply:
yes.  For `type StructuredTag`, I had to put back `String` as the type for namespace, key, and value; leaving the rest as in the new form.   Now the graphql server starts.  Basically, I need to learn GraphQL, apollo, resoolvers, and generator, inputs and outputs

Thanks for the tip.